lib/tasks/ont_data.rake

```diff
@@ -46,18 +65,33 @@ end
 def create_plates(count:)
   puts
   puts "-> Creating #{count} ONT plates"
+
+  plate_no = Plate&.last&.id || 0
+
+  barcodes = create_number_of_plates(count, plate_no)
+
+  puts '-> Successfully created ONT plates'
+  barcodes
+end
+
+def create_number_of_plates(count, plate_no)
   variables = OntPlates::Variables.new
   constants_accessor = Pipelines::ConstantsAccessor.new(Pipelines.ont.covid)
-
-  count.times do |i|
-    plate_no = i + 1
-    submit_create_plate_query(plate_no: plate_no,
-                              barcode: "DEMO-PLATE-#{plate_no}",
-                              wells: variables.wells(sample_name: "for Demo Plate #{plate_no}",
-                                                     constants_accessor: constants_accessor))
+  [].tap do |barcodes|
```

Comment:
I suspect this block could be written more clearly as:

```
barcodes = (0..count).map do |i|
  create_plate(i + 1, variables, constants_accessor)
end
```

Reply:
I have done it slightly differently but pretty similar.